src/components/header.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const Header = () => (
+  <div>
+    <header className="header">
+      <h1 className="header-heading">Bookstore CMS</h1>
+      <nav>
+        <Link to="/" className="header-primary-heading">BOOKS</Link>
+        <Link to="/category" className="header-secondary-heading">CATEGORIES</Link>
```

Comment:
✅ Has 2 `<Link>`s in the navbar which leads to the booklist and categories locations 👍🏻.